src/controller/controller.ts

```diff
@@ -76,11 +83,16 @@ export function login(req: Request, res: Response) {
     }
 }
 
-export function signup(req: Request, res: Response) {
+export async function signup(req: Request, res: Response) {
```

Comment:
did adding your functions make the login and sign up functions async?

Reply:
I had to add the async because of the `await validateInput()` but I don't think it affects anything else. I could be wrong though!